File: tests/test_game_sim.py
import context

from copy import deepcopy
import json

from bbgm.core import game_sim

class GameSimTestSuite():
    """Game simulation test cases."""

    def test_game(self, team1, team2):
        gs = game_sim.GameSim(team1, team2)
        return gs.run()

    def test_games(self):
        pass
#        for i in xrange(100):
#            self.test_game()

if __name__ == '__main__':
    f = open('teams.json')
    teams = json.load(f)
    f.close()

    t = GameSimTestSuite()
    print json.dumps(t.test_game(teams[0], teams[1]))

File: tests/test_game_sim.js
fs = require('fs');
teams = JSON.parse(fs.readFileSync('teams.json').toString());
//console.log(teams);
game_sim = require('../bbgm/static/js/game_sim');
//console.log(game_sim);
gs = new game_sim.GameSim(teams[0], teams[1]);
//console.log(gs);
console.log(JSON.stringify(gs.run()));
